oe_paragraphs.install

```diff
@@ -183,3 +183,40 @@ function oe_paragraphs_update_8001(array &$sandbox): void {
     }
   }
 }
+
+/**
+ * Update the content translation settings of the existing paragraphs types.
+ *
+ * We need to update all paragraph types to hide the non-translatable fields on
```

Comment:
Since the phpdoc is used to display the pending updates, let's move this inside the function itself.